package.json

```diff
@@ -22,5 +24,6 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "5.2.0",
     "nodemon": "2.0.15"
-  }
+  },
+  "type": "module"
```

Comment:
To byna npisał ponad dependencies property

Reply:
zrobione